projects/core/src/features-config/feature-toggles/config/feature-toggles.ts

```diff
@@ -649,6 +649,11 @@ export interface FeatureTogglesInterface {
    */
   a11yViewHoursButtonIconContrast?: boolean;
 
+  /**
+   * `Cart component` improve action buttons appearance
+   */
+  a11yImproveCardActionButtons?: boolean;
```

Comment:
[We should make this description more accurate](https://wiki.one.int.sap/wiki/display/spar/Coding+Guidelines#CodingGuidelines-WriteaJSDocdescriptionofaFeatureToggle:~:text=non%2Dvoid%20elements.-,2.18.%20Use%20Feature%20Toggles%20when%20changing%20behavior%20of%20an%20existing%20feature,-When%20introducing%20breaking). The changes to my-preferred-store.component.html feel significant enough